var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -888,7 +888,8 @@ def fix_package_config(self):
             content += " " + self.spec["lapack"].libs.ld_flags
             content += " " + self.spec["fftw-api"].libs.ld_flags
 
-            if (self.spec["fftw-api"].name == "fftw") and ("+openmp" in self.spec["fftw"]):
+            fftw = self.spec["fftw-api"]
```

Comment:
Why is this needed, given the definition at line 367 ?

Reply:
Oh, lol, my bad! From the web-gui I didn't notice Github was squeezing 500 lines in between :laughing: